.ci/tm-tests-playground

```diff
@@ -84,6 +84,9 @@ echo "Testmachinery landscape: ${TM_LANDSCAPE}"
 echo "Arguments: ${ARGUMENTS}"
 echo "EFFECTIVE_VERSION: ${EFFECTIVE_VERSION}"
 
+GIT_REVISION="${EFFECTIVE_VERSION##*\-}"
```

Comment:
Could you please explain what this is about?

Reply:
This operator splits the EFFECTIVE_VERSION string which looks like `v0.13.0-dev-<git_commit_hash>` and gets the last element.